src/components/attendee/RetunableMediaElement/hooks.tsx

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from "react";
+
+import { COLLECTION_RETUNABLE_MEDIA_ELEMENTS } from "settings";
+
+import { useRefiDocument } from "hooks/fire/useRefiDocument";
+
+import {
+  NotTunedSettings,
+  RetunableMediaElementSettings,
+  RetunableMediaSource,
+} from "./types";
+
+interface useRetunableMediaElementDataOptions {
+  spaceId: string;
+}
+
+const NOT_TUNED_SETTINGS: NotTunedSettings = Object.freeze({
+  sourceType: RetunableMediaSource.notTuned,
+});
+
+export const useRetunableMediaElement = ({
+  spaceId,
+}: useRetunableMediaElementDataOptions) => {
+  const queryPath = useMemo(
+    () => [COLLECTION_RETUNABLE_MEDIA_ELEMENTS, spaceId],
+    [spaceId]
+  );
+
+  const {
+    data: settings,
+    isLoading,
+  } = useRefiDocument<RetunableMediaElementSettings>(queryPath);
```

Comment:
Please try to replace it with `useFireDocument` to lessen the dependence of Reactfire. The alternative should also help with missing IDs by way of `DEFERRED`.

In case of problems with the suggested hook, please just add `@debt` comment here.

Reply:
Resolved in 40c9750b0